### ISSUE TITLE
Preparation for the Redesign

### ISSUE THREAD

**Issue by yatil** [w3c/wai-tutorials#545]
👋 The redesign is coming to this repository soon. (Hooray! 🎉) We need to check off several steps before we can link your content to the [redesign repository](https://github.com/w3c/wai-website). I marked some of the steps with RM meaning Resource Manager. Others are marked with @yatil, and I’ll do/help you through the technical parts.

In addition to this issue, you might find other issues in your repository that are labeled with wai-redesign-before that should be addressed/resolved before the redesign. That might mean to postpone working on it until after the redesign (use the label wai-redesign-after for those issues).

* [ ] **Content** is updated and ready to be put into the new design. (RM)
  * The best case scenario would be that the content is ready before doing step three below. However, that might not be feasible in every case, so @yatil might do the next step before the content is ready and just port the content that is available to him.
  * Once done, add label wai-redesign-content-done.
* [ ] **Tech** Create branch/repository with the new structure (@yatil)
  * For simple resources (separate pages/pages that don’t need a left navigation), you will find the documents in the root directory of the repository.
  * For more complicated resources, @yatil creates a Jekyll collection which adds the left sidebar navigation amongst other things. You will then find your files in a folder called `_resource_name`. 
   * Documentation should be available in the [w3c/wai-website wiki](https://github.com/w3c/wai-website/wiki/Repository-Anatomy-for-individual-resources) soon.
  * Once done, add label wai-redesign-repo-done.
* [ ] **Content** Use current content and port it to the new design (@yatil)
  * Once this is done, all previews will be in the new design.
  * Once done, add label wai-redesign-content-ported.
* [ ] **Content** Check if content renders well, make changes as needed/wanted. ([documentation](https://github.com/w3c/wai-website/wiki/Preparing-Content)) (RM)
  * Once done, add label wai-redesign-content-checked.
* [ ] **Tech** Link this repository to the w3c/wai-website repository (@yatil)
  * Once done, add label wai-redesign-repo-linked.
* When all checkpoints above are checked, close this issue.
